assets/css/style.css

```diff
@@ -331,15 +377,15 @@
   }
 
   .about {
```

Comment:
- [ ] Similarly, for the about section, an empty space is present on right side and the technologies are overflowing⚠😓 Kindly fix this as well to fulfil the project requirements as mentioned above 👆

![image](https://github.com/Graciano1997/portofolio-graciano/assets/106119105/24f297ce-68f7-4952-a676-575beccfeecd)

- [ ] For frameworks and skills, what do you think of adding some space on left and right side? 😄 Can you please add it to get a design close to the Figma template according to the project requirements? ✨

Your design | Given design
--| ---|
![image](https://github.com/Graciano1997/portofolio-graciano/assets/106119105/4e922eee-ddb0-4e35-bd0d-1a19b6603104) | ![image](https://github.com/Graciano1997/portofolio-graciano/assets/106119105/39e50255-fce7-4f8b-ac96-84e803f01a38)